src/object.c

```diff
@@ -1627,12 +1631,24 @@ NULL
         addReplyBulkCString(c,"allocator.resident");
         addReplyLongLong(c,server.cron_malloc_stats.allocator_resident);
 
+        addReplyBulkCString(c,"allocator.retained");
+        addReplyLongLong(c,server.cron_malloc_stats.allocator_retained);
```

Comment:
btw, i'm not sure it's interesting to expose this metric. it doesn't really affect anything.

btw we did want to consider exposing breakdown of memory usage to small and large bins, but i'm not sure that'll matter to anyone, and whoever does care, will probably extract the full jemalloc stats.

Reply:
it's necessary if these metrics are useful, not all users knows how to use the `memory malloc-stats` to get theses metric from jemalloc output which is still a bit complicated.

Reply:
yes, but `retained` isn't useful for anything. (no one really cares about vmsize either).
so i think if we start exposing both `regained` and `muzzy`, it can cause confusing (i was confused in the past), and people will get them mixed up.
if if don't think it's useful, let's drop it.

Reply:
already removed.